autoortho/getortho.py

```diff
@@ -688,18 +691,20 @@ def get_img(self, mipmap, startrow=0, endrow=None, maxwait=5, min_zoom=None):
         #
 
         # Get effective zoom
-        zoom = self.zoom - mipmap
+        gzo_effective = min(self.global_zoom_out, max(0, 4 - mipmap))
+        zoom = self.zoom - (mipmap + gzo_effective)
+
         log.debug(f"GET_IMG: Default zoom: {self.zoom}, Requested Mipmap: {mipmap}, Requested mipmap zoom: {zoom}")
         col, row, width, height, zoom, mipmap = self._get_quick_zoom(zoom, min_zoom)
         log.debug(f"Will use:  Zoom: {zoom},  Mipmap: {mipmap}")
         
         # Do we already have this img?
-        if mipmap in self.imgs:
-            log.debug(f"GET_IMG: Found saved image: {self.imgs[mipmap]}")
-            return self.imgs.get(mipmap)
+        if mipmap - gzo_effective in self.imgs:
```

Comment:
By setting this up in _get_quick_zoom, all this manipulation can likely go away.